src/components/BlacklistedTagsSelector/index.tsx

```diff
@@ -0,0 +1,438 @@
+import Tooltip from '@app/components/Common/Tooltip';
+import { encodeURIExtraParams } from '@app/hooks/useDiscover';
+import defineMessages from '@app/utils/defineMessages';
+import { Transition } from '@headlessui/react';
+import {
+  ArrowDownIcon,
+  ClipboardDocumentIcon,
+} from '@heroicons/react/24/solid';
+import type { TmdbKeywordSearchResponse } from '@server/api/themoviedb/interfaces';
+import type { Keyword } from '@server/models/common';
+import { useFormikContext } from 'formik';
+import {
+  forwardRef,
+  useCallback,
+  useEffect,
+  useImperativeHandle,
+  useRef,
+  useState,
+} from 'react';
+import { useIntl } from 'react-intl';
+import type { ClearIndicatorProps, GroupBase, MultiValue } from 'react-select';
+import { components } from 'react-select';
+import AsyncSelect from 'react-select/async';
+import { useToasts } from 'react-toast-notifications';
+import useClipboard from 'react-use-clipboard';
+
+const messages = defineMessages('components.Settings', {
+  copyBlacklistedTags: 'Copied blacklisted tags to clipboard.',
+  copyBlacklistedTagsTip: 'Copy blacklisted tags configuration',
+  importBlacklistedTagsTip: 'Import blacklisted tag configuration',
+  clearBlacklistedTagsConfirm:
+    'Are you sure you want to clear the blacklisted tags?',
+  yes: 'Yes',
+  no: 'No',
+  searchKeywords: 'Search keywords…',
+  starttyping: 'Starting typing to search.',
+  nooptions: 'No results.',
+  blacklistedTagImportTitle: 'Import Blacklisted Tag Configuration',
+  blacklistedTagImportInstructions: 'Paste blacklist tag configuration below.',
+  valueRequired: 'You must provide a value.',
+  noSpecialCharacters:
+    'Configuration must be a comma delimited list of TMDB keyword ids, and must not start or end with a comma.',
+  invalidKeyword: '{keywordId} is not a TMDB keyword.',
+});
+
+type SingleVal = {
+  label: string;
+  value: number;
+};
+
+type BlacklistedTagsSelectorProps = {
+  defaultValue?: string;
+};
+
+const BlacklistedTagsSelector = ({
+  defaultValue,
+}: BlacklistedTagsSelectorProps) => {
+  const { setFieldValue } = useFormikContext();
+  const [value, setValue] = useState<string | undefined>(defaultValue);
+  const [selectorValue, setSelectorValue] =
+    useState<MultiValue<SingleVal> | null>(null);
+
+  const update = useCallback(
+    (value: MultiValue<SingleVal> | null) => {
+      const strVal = value?.map((v) => v.value).join(',');
+      setSelectorValue(value);
+      setValue(strVal);
+      setFieldValue('blacklistedTags', strVal);
+    },
+    [setSelectorValue, setValue, setFieldValue]
+  );
+
+  return (
+    <>
+      <ControlledKeywordSelector
+        value={selectorValue}
+        onChange={update}
+        defaultValue={defaultValue}
+        components={{
+          DropdownIndicator: undefined,
+          IndicatorSeparator: undefined,
+          ClearIndicator: VerifyClearIndicator,
+        }}
+      />
+
+      <BlacklistedTagsCopyButton value={value ?? ''} />
```

Comment:
You could maybe disable the button if the value is empty?

Reply:
Done in [edc1802](https://github.com/fallenbagel/jellyseerr/pull/1306/commits/edc1802b3d793d6c4d795bc7190b231b41c550f0)

Reply:
Good idea, added general CSS support for disabled button.input-actions too